public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -770,15 +790,19 @@ function saveBookmark() {
         sessionStorage.setItem("Bookmark", URL);
         createBookmarks();
     }
-    else {
-        console.log("one bookmark");
-    }
 }
 function createBookmarks() {
     const LI = document.createElement("li");
     const URL = sessionStorage.getItem("Bookmark");
     const TITLE = sessionStorage.getItem("Title");
```

Comment:
Just as an FYI, you can store complex data in session/local storage such as objects and arrays. If you ever need to store lots of items these structures can do the job with I believe JSON.stringify and JSON.parse.

Reply:
will def use json once we do mulitple bookmarks